src/openrct2/ride/RideData.h

```diff
@@ -436,6 +436,7 @@ enum class RtdFlag : uint8_t
     isMaze = 58,
     isSpiralSlide = 59,
     allowReversedTrains = 60,
+    isMiniGolf = 61,
```

Comment:
I'm not sure about adding more flags like this, a ride descriptor shouldn't be able to claim to be a maze and cash machine at the same time. Why doesn't the ride descriptor have something like ride archetype/type? Those flags should only describe what a ride is capable of but not describe what it is.

Reply:
That’s an interesting idea, actually. We could just have an enum for these special types. I guess you prefer me doing that in this PR, rather than a followup?